src/player.cpp

```diff
@@ -11321,12 +11321,10 @@ int player::visibility( bool, int ) const
     if ( is_invisible() ) {
         return 0;
     }
-    if( has_trait( trait_CRAFTY ) ) {
-        return 60;
-    }
     // @todo:
     // if ( dark_clothing() && light check ...
-    return 100;
+    int stealth_modifier = std::floor( mutation_value( "stealth_modifier" ) );
+    return clamp( 100 - stealth_modifier, 60, 160 );
```

Comment:
`clamp( x, 60, 160 )` is 110 +/- 50. Is this as intended?

Reply:
It wasn't, I made a blunder; fixed now

Thanks